pkgs/applications/graphics/inkscape/extensions/textext.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, fetchFromGitHub
+, inkscape
+, texlive
+, python3
+}:
+
+let
+  texlive-combined = texlive.combined.scheme-basic;
```

Comment:
Does the extension support arbitrary latex packages? In that case this should probably be passed as an argument so it can be changed using `override`?